### PR TITLE
Add color section to styleguide

### PR DESCRIPTION
**Pull request by jimbo**
## Description
Add a color section to the styleguide. The prose content may be lorem ipsum, but the values should reflect the official Venia styles.

## Related Issue
PWA-224

## Acceptance 
- @jcalcaben should affirm document structure
- @soumya-ashok should affirm Venia typographic styles
- Others may review the code

### Verification Stakeholders
- @jcalcaben 
- @soumya-ashok 

### Verification Steps

1. `yarn workspace @magento/venia-styleguide run start`
2. Navigate to `/page/color`

## Checklist
* I have updated the documentation accordingly, if necessary.
* I have *not* added tests to cover my changes, if necessary.

### CODE REVIEW

**Comment by dpatil-magento** [comment]
Verification steps and regression pass but need @jcalcaben  and @soumya-ashok review to get this merged.

File: packages/venia-styleguide/src/components/Palette/Tone.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    return factor <= 0.03928
+        ? factor / 12.92
+        : Math.pow((factor + 0.055) / 1.055, 2.4);
+};
+
+const getLightness = channels => {
+    const [r, g, b] = channels.map(getSRGB);
+
+    return r * 0.2126 + g * 0.7152 + b * 0.0722;
```
Comment (revanth0212) [inline packages/venia-styleguide/src/components/Palette/Tone.js]: Can you drop a comment about these numbers?

File: packages/venia-styleguide/src/components/Palette/Tone.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import classes from './Tone.css';
+
+const getHex = channel => parseInt(channel).toString(16);
+
+const getSRGB = channel => {
+    const factor = channel / 255;
+
+    return factor <= 0.03928
+        ? factor / 12.92
+        : Math.pow((factor + 0.055) / 1.055, 2.4);
```
Comment (revanth0212) [inline packages/venia-styleguide/src/components/Palette/Tone.js]: Can you drop a comment about these numbers?

**Comment by jimbo** [comment]
> Couple questions around numbers used. Dropping some comments will help.

Thanks, good catch. I've extracted those utils to a new file and put a comment in there citing the original source. The numbers you highlighted are magic numbers from a formula that I obviously didn't create.

**Comment by jimbo** [comment]
Addressed feedback from @soumya-ashok by putting the colors into sections that reflect their usage, and ordering those sections.

**Pull request merged**